File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/DocumentFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Document;

public class DocumentFilterValidator extends FilterValidator<Document> {

    public DocumentFilterValidator(CustomValidator<Document> validator) {
        super(Document.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/DocumentFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Document
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class DocumentFilterValidator(validator: Validator<Document>) : FilterValidator<Document>(Document::class, validator)

